packages/app-layout/src/AppLayout.stories.tsx

```diff
@@ -1,45 +1,46 @@
-import { ComponentMeta, ComponentStory } from '@storybook/react';
 import { Prose } from '@ag.ds-next/react/prose';
```

Comment:
Upgraded our stories to [Component Story Format 3.0](https://storybook.js.org/blog/component-story-format-3-0/)